polyfills/Promise/config.json

```diff
@@ -14,7 +14,7 @@
 		"bb": "* - 10",
 		"chrome": "* - 31",
 		"firefox": "6 - 28",
-		"ie": "8 - *",
+		"ie": "< 12",
```

Comment:
D'oh, I guess I misunderstood what you're asking for. I had it as `< 12` before, but if the polyfill just doesn't work in IE7 (and Promises certainly don't exist in IE7), then what's the proper solution?